Cargo.toml

```
@@ -28,14 +28,14 @@ features = [
 calloop = { version = "0.12.3", features = ["signals"] }
 catacomb_ipc = { version = "0.1.0", path = "./catacomb_ipc", features = ["clap", "smithay"] }
 clap = { version = "4.2.3", features = ["derive", "wrap_help"] }
-dirs = "4.0.0"
+dirs = "5.0.0"
 libc = "0.2.123"
-nix = "0.26.2"
+nix = "0.27"
 serde_json = "1.0.85"
 serde = { version = "1.0.144", features = ["derive"] }
 tokio = "1.26.0"
 tracing = "0.1.37"
 tracing-subscriber = { version = "0.3.17", features = ["env-filter"] }
-udev = "0.6.2"
+udev = "0.8"
```

```suggestion
udev = "0.8.0"
```

Always specify the full versions to properly indicate precision. This applies to all changes in this PR.